packages/migrate/src/MigrateEngine.ts

```diff
@@ -296,6 +324,8 @@ export class MigrateEngine {
           },
         })
 
+        this.isRunning = true
```

Comment:
Now way that the code after this still makes it fail in some way that `isRunning=true` would be inaccurate?

Reply:
This `isRunning` properties is implemented mirroring the same property in the IntrospectionEngine, and is updated in the same way. The idea is to first get the MigrateEngine on par with the features exposed by the IntrospectionEngine we need to make use of, and later verify possible improvements that could be done